### PR TITLE
nvme-cli: 2.1 -> 2.2.1 and libnvme: 1.1 -> 1.2

### PR DESCRIPTION
**Pull request by ifurther**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/os-specific/linux/nvme-cli/default.nix [hunk 1]
```diff
@@ -15,7 +15,7 @@ stdenv.mkDerivation rec {
     owner = "linux-nvme";
     repo = "nvme-cli";
     rev = "v${version}";
-    hash = "sha256-15mmyrhh0mc9symm57nrw9gbl2fwif1jd9pan708f9n3wq09l36p=";
+    sha256 = "sha256-15mmyrhh0mc9symm57nrw9gbl2fwif1jd9pan708f9n3wq09l36p=";
```
Comment (ajs124) [inline pkgs/os-specific/linux/nvme-cli/default.nix:18]: why?

Comment (ifurther) [reply pkgs/os-specific/linux/nvme-cli/default.nix]: I rollback it.

**Comment by ajs124** [comment]
The changelog says `libuuid` isn't necessary anymore, but this still fails to build for [and ofborg](https://logs.nix.ci/?key=nixos/nixpkgs.198471&attempt_id=cb012d11-46b7-4d91-ba05-bc52f92e8e5e) saying `fatal error: uuid/uuid.h: No such file or directory`.

**Comment by zseri** [comment]
@ajs124 can you report an upstream issue for it, please?

**Comment by ajs124** [comment]
@zseri see https://github.com/linux-nvme/nvme-cli/pull/1715

**Pull request merged**